### DISCUSSION TITLE
Warning message about mei/i2c/i2c-dev probe/add failed after ced79fce4a686b

### DISCUSSION THREAD

**Discussion by hugh712** [fwupd/fwupd#4810] [General]
Hi, 

After below commit, I saw some "Warning messages"[0] on my laptop,
seems it's because the program tried to probe/add new added mei/i2c/i2c-dev nodes,
so the things I would like to discuss is:

If it's just "try to probe", can we use debug level instead of warning level in the log?  

_ced79fce4a686bd06fe6f69e305fb839b3b61db5 (HEAD)
Author: Richard Hughes <richard@hughsie.com>
Date:   Thu Jun 30 15:53:29 2022 +0100    Create the better device GType in the backend
 
It is weird to do this when constructing the object, and it allows us
to match PossiblePlugin quirk matches on potentially better instance
IDs added by the superclass._

[0]
failed to probe device /sys/devices/pci0000:00/0000:00:15.0/i2c_designware.0/i2c-1/i2c-WCOM49B2:00: cannot parse WCOM49B2:00
failed to probe device /sys/devices/pci0000:00/0000:00:15.1/i2c_designware.1/i2c-2/i2c-DLL0945:00: cannot parse DLL0945:00
failed to probe device /sys/devices/pci0000:00/0000:00:15.1/i2c_designware.1/i2c-2/i2c-VEN_04F3:00: cannot parse VEN_04F3:00
failed to add device /sys/devices/pci0000:00/0000:00:16.0/mei/mei0: ioctl error: Inappropriate ioctl for device

**Comment by hugh712** [comment]
not sure it's proper or not just change warning to debug in [0]

[0] https://github.com/fwupd/fwupd/blob/main/src/fu-engine.c#L6606

**Comment by hughsie** [comment]
Fixed in https://github.com/fwupd/fwupd/pull/4813

**Reply by hugh712** [reply]
cool, thank you :)

**Reply by hugh712** [reply]
after the verification, the warning messages are gone, thank you
